doc/reference/bluetooth/hci.txt

```diff
@@ -42,7 +42,8 @@ command is supported.
 	|       | 4    | Set Scan Request Reports                   |
 	|       | 5    | Write Tx Power Level (per Role/Connection) |
 	|       | 6    | Read Tx Power Level (per Role/Connection)  |
-	|       | 7    | Reserved                                   |
+	|       | 7    | Get USB Mode                               |
+	|       | 7    | Set USB Mode                               |
```

Comment:
Initially it was made independent of USB but @holtmann suggested it to be just for USB since so far is the only transport known to have races, the read was introduce more for the sake of completeness.

I will fix the bits though.